views-core/src/test/groovy/io/micronaut/views/ViewUtilsSpec.groovy

```diff
@@ -0,0 +1,19 @@
+package io.micronaut.views
+
+import spock.lang.Specification
+
+class ViewUtilsSpec extends Specification {
+    def "normalizeFile handles extensions with and without ."(path, extension, expected) {
```

Comment:
Can you annotate the spec with [@Unroll](https://spockframework.org/spock/javadoc/1.0/spock/lang/Unroll.html)

Reply:
Done.

Reply:
I've added string interpolation to unroll so that the reports look like: 
<img width="634" alt="Screenshot 2021-12-10 at 16 42 10" src="https://user-images.githubusercontent.com/864788/145600926-612a1319-b934-4cb4-9434-10c016b1b022.png">

Reply:
I see. Thanks for your help.